### PR TITLE
[CBRD-24063] Add new java file for testing javasp of cbrd-24063

### PR DESCRIPTION
**Pull request by zionyun**
refer to http://jira.cubrid.org/browse/CBRD-24063

### CODE REVIEW

**Review by ssihil — APPROVED** [review]

File: CTP/sql/function/stored_procedure/src/SpTest8.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                        pstmt = conn.prepareStatement("select class_name from db_class");
+                        rs = pstmt.executeQuery();
+                        if (rs.next()) {
+                                sql = rs.getString("class_name");
+                        }
+                        rs.close();
+                        pstmt.close();
+                        conn.close();                
+                } catch (Exception e) {
+                        e.printStackTrace();
```
Comment (hgryoo) [inline CTP/sql/function/stored_procedure/src/SpTest8.java]: remove e.printStackTrace()

File: CTP/sql/function/stored_procedure/src/SpTest8.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                        pstmt = conn.prepareStatement("select class_name from db_class");
+                        rs = pstmt.executeQuery();
+                        if (rs.next()) {
+                                sql = rs.getString("class_name");
+                        }
+                        rs.close();
+                        pstmt.close();
+                        conn.close();
+                } catch (Exception e) {
+                        e.printStackTrace();
```
Comment (hgryoo) [inline CTP/sql/function/stored_procedure/src/SpTest8.java]: remove e.printStackTrace();

**Review by hgryoo — APPROVED** [review]

File: CTP/sql/function/stored_procedure/src/SpTest8.java (hunk 1 above)
Comment (zionyun) [reply CTP/sql/function/stored_procedure/src/SpTest8.java]: Modifyed to fit the format like other SpTest code.

File: CTP/sql/function/stored_procedure/src/SpTest8.java (hunk 2 above)
Comment (zionyun) [reply CTP/sql/function/stored_procedure/src/SpTest8.java]: Modifyed to fit the format like other SpTest code.

**Review by ssihil — APPROVED** [review]

**Review by sy052 — APPROVED** [review]

**Pull request merged**